buildtools/pipenv/pipenv.go

```diff
@@ -0,0 +1,148 @@
+package pipenv
+
+import (
+	"encoding/json"
+
+	"github.com/fossas/fossa-cli/errors"
+	"github.com/fossas/fossa-cli/exec"
+	"github.com/fossas/fossa-cli/graph"
+	"github.com/fossas/fossa-cli/pkg"
+)
+
+// Pipenv defines the interface for all pipenv tool implementation.
+type Pipenv interface {
+	Deps() (graph.Deps, error)
+}
+
+// PipenvCmd implements Pipenv by parsing command output.
+type PipenvCmd struct {
```

Comment:
[nit] Now that I think about it, this might be better named `Cmd` because `pipenv.PipenvCmd` stutters.